admin/applications/__init__.py

```diff
@@ -99,9 +99,11 @@ def download(reference_number):
         db.session.commit()
 
         from grc.utils.application_files import ApplicationFiles
+        application_data = application.application_data()
+        application_data.updated = application.updated
```

Comment:
The `application_data.updated` property will only be set when the PDF bundle is generated _**via the admin portal**_.

Usually, the PDF bundle is generated _**via the GRC application**_ when the user clicks `Submit application`.
So, in most cases, the updated date won't be shown in the bundle.

I've raised a PR with a suggested change to fix this: https://github.com/cabinetoffice/grc-app/pull/169